blocks/subscriptions-block/features/SubscriptionProfileManagement/children/BasicSubscriptionDetail/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, {useState, useEffect} from "react";
+import { usePhrases, Stack, Button, Badge, Icon, Paragraph } from "@wpmedia/arc-themes-components";
+
+const ACTIVE = "Active";
+const CANCELED = "Canceled";
+const TERMINATED = "Terminated";
+
+const getStatus = (sub) => {
+  if(sub?.status === 1) return ACTIVE;
```

Comment:
Hi @edwardcho1231 , do you mind include as "ACTIVE" to those subs with status: 'Suspended', 'Vindicia', 'Dunning'. All these are considered as active too.

Reply:
@LauraPinilla is gift sub considered active?